internal/provider/resources/install/aws/aws_iam_write.go

```diff
@@ -0,0 +1,400 @@
+// Copyright (c) 2024 P0 Security, Inc
+// SPDX-License-Identifier: MPL-2.0
+
+package installaws
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/objectvalidator"
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringdefault"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+	"github.com/p0-security/terraform-provider-p0/internal"
+	installresources "github.com/p0-security/terraform-provider-p0/internal/provider/resources/install"
+)
+
+// Ensure provider defined types fully satisfy framework interfaces.
+var _ resource.Resource = &AwsIamWrite{}
+var _ resource.ResourceWithImportState = &AwsIamWrite{}
```

Comment:
Since we're using Configure, we should also include `var _ resource.ResourceWithConfigure = &AwsIamWrite{}`

Reply:
ack